### PR TITLE
Improve the box styles according to the specs

### PR DESCRIPTION
**Pull request by Ladsgroup**
- Improve responsiveness by removing the extra padding when the boxes
   are stacking on top of each other.
 - Add border based on the specs

### CODE REVIEW

**Comment by guergana** [comment]
Hi, Amir, where are the specs? I don't see the gray border in figma.

File: src/components/QueryBuilder.vue [hunk 1]
```diff
@@ -123,7 +123,8 @@ a {
 	margin-block: $dimension-layout-xsmall;
 	padding-block: $dimension-layout-xsmall;
 	padding-inline: $dimension-layout-xsmall;
-	background-color: $color-base-70; // maybe replace with an alias token?
+	background-color: $color-base-80; // maybe replace with an alias token?
+	border: $border-width-thin $border-style-base $border-color-base-default;
```
Comment (micgro42) [inline src/components/QueryBuilder.vue:127]: I think they still need the `$border-radius-base`

**Review by micgro42 — CHANGES_REQUESTED** [review]
Thanks for those adjustments! Just the border radius is missing, I think :)

The specs have been added to here https://www.figma.com/file/hZpaYX0xNNhK0L7kBvN6WK/Query-Builder?node-id=2604%3A29420

**Comment by Ladsgroup** [comment]
> Thanks for those adjustments! Just the border radius is missing, I think :)
> 
> The specs have been added to here https://www.figma.com/file/hZpaYX0xNNhK0L7kBvN6WK/Query-Builder?node-id=2604%3A29420

Oh thanks. They didn't exist in the CSS but they are added as a note (that's why I missed that)
image

**Review by micgro42 — APPROVED** [review]
Ready to go :)

**Pull request merged**